include/nuttx/arch.h

```diff
@@ -2398,6 +2398,40 @@ int up_rtc_getdatetime_with_subseconds(FAR struct tm *tp, FAR long *nsec);
 int up_rtc_settime(FAR const struct timespec *tp);
 #endif
 
+/****************************************************************************
+ * Name: up_tls_size
+ *
+ * Description:
+ *   Get tls (tdata + tbss) section size
+ *
+ * Returned Value:
+ *   Size of (tdata + tbss)
+ *
+ ****************************************************************************/
+
+#ifdef CONFIG_ARCH_TLS
+int up_tls_size(void);
```

Comment:
need move to?
./arch/risc-v/include/tls.h
./arch/hc/include/tls.h
./arch/z80/include/tls.h
./arch/misoc/include/tls.h
./arch/x86/include/tls.h
./arch/sim/include/tls.h
./arch/avr/include/tls.h
./arch/renesas/include/tls.h
./arch/or1k/include/tls.h
./arch/xtensa/include/tls.h
./arch/x86_64/include/tls.h
./arch/arm/include/tls.h
./arch/mips/include/tls.h
./arch/z16/include/tls.h

Reply:
Shouw we dup this declaration in these files ?

Reply:
Ok, I provide a patch to remove the dup: https://github.com/apache/incubator-nuttx/pull/4941